index.html

```diff
@@ -0,0 +1,168 @@
+                    </div>
+                    <div class="program_in">
+                        <div> <img src="./images/icon_02.png" alt="title" /></div>
+                        <p>Exhibition</p>
+                        <h4>Exhibition of works in different finishes by Colombian leads who have participated in
+                            different
+                            fairs around the world.
+                        </h4>
+
+                    </div>
+                    <div class="program_in">
+                        <div><img src="./images/icon_03.png" alt="title" /></div>
+                        <p>Forums</p>
+                        <h4> Designers from various contries speaking about their experiences and new trends.</h4>
+                    </div>
+                    <div class="program_in">
+                        <div><img src="./images/icon_04.png" alt="title" /></div>
+                        <p>Worshop</p>
+                        <h4> We will do a practical workshop, where we will explore the most used techniques for design
+                            thinking.</h4>
+                    </div>
+                    <div class="program_in">
+                        <div><img src="./images/icon_05.png" alt="title" /></div>
+                        <p>CHPT Feedback</p>
+                        <h4> We will make a small feedback in figures of these years of activity and how the design
+                            movement
+                            has grown and we will see how it has grown.</h4>
+                    </div>
+                </div>
+
+                <h2>SEE THE WHOLE PROGRAM</h2>
+
+            </div>
+        </section>
+
+        <section class="speaker_heros" id="speakers">
+            <div class="container">
+                <div class="centered-title">
+                    <h2 class="black-title">Featured Heros</h2>
+                    <hr>
+                </div>
+
+
+                <div class="heros-container"></div>
+
+            </div>
+
+        </section>
+
+        <section id="sponsors" class="sponsors">
+            <div class="container sponsors">
+                <h2 class="white-tittle">Partners</h2>
+                <hr>
+                <div class="sponsor-row">
+                    <div><img src="./images/google.png" alt="logo" /></div>
+                    <div><img src="./images/autodesk.png" alt="logo" /></div>
+                    <div><img src="./images/avengers.png" alt="logo" /></div>
+                    <div><img src="./images/makita.png" alt="logo" /></div>
+                    <div><img src="./images/stark.png" alt="logo" /></div>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <div class="container-header-footer">
+        <div class="footer-home">
+            <div><img src="./images/logo_bn_bogota.png" alt="logo" /></div>
+            <p>Bogotá City 2022 Copy right reserved</p>
+        </div>
+</div>
```

Comment:
- [ ] As the previous reviewer suggested, please ensure to use semantic tags. In this case, please consider using  `footer` tag

Reply:
if you see the footer are diferent, into the CSS this made conflict for that i decided use a div